scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
However, I've found that this:
```
sudo ln -s /dev/null /etc/udev/rules.d/75-persistent-net-generator.rules
```
solves the problem.

Source: https://askubuntu.com/questions/628217/use-of-predictable-network-interface-names-with-alternate-kernels  -- at the bottom of the accepted answer.